### ISSUE TITLE
[BUG] Unable to calculate relative force deviation.

### ISSUE THREAD

**Issue by xiaohe-njust** [deepmodeling/dpgen#1639]
### Bug summary

When using dpgen for structural exploration, if an external LAMMPS template is used, the relative force model deviation cannot be calculated, and the output model_devi.out file information is the absolute force deviation. When no external LAMMPS template is used, everything is normal.

### DP-GEN Version

0.12.1

### Input Files, Running Commands, Error Log, etc.

[param.json](https://github.com/user-attachments/files/17037364/param.json)

`	"use_relative": true,
	"epsilon": 1.0,`
If an external LAMMPS template is used, regardless of whether the code for calculating relative force deviation is used, the output model_devi.out file information remains the same. Trying to change the value of the keyword epsilon still has no effect. When no external LAMMPS template is used, everything is normal.

### Steps to Reproduce

[input.zip](https://github.com/user-attachments/files/17037404/input.zip)

**Comment by njzjz** [comment]
This is actually the same problem as #1608

**Comment by cheng2t** [comment]
Hello, I also encountered this problem, can you please solve it？

**Comment by cheng2t** [comment]
Hello, I'm also experiencing this problem, have you solved it yet?

**Comment by xiaohe-njust** [comment]
> Hello, I'm also experiencing this problem, have you solved it yet?

No, it's not settled.

**Comment by njzjz** [comment]
Hi @cheng2t, we don't have enough people to maintain this project. If you are willing to contribute to the project, don't hesitate to do it!

**Comment by cheng2t** [comment]
Thanks for your reply，Do you have any problems with the phenomenon that the accuracy rate has not been improved during the iteration process?

**Comment by xiaohe-njust** [comment]
@cheng2t Yes, I encountered this issue and discovered this bug. Incorrect use of absolute force deviation can lead to the newly added structures in each iteration being unreasonable. If your configuration involves large force values, you can try increasing the value of model_devi_f_trust_lo. The numerical setting for relative force deviation is not suitable for setting absolute force deviation.

**Comment by cheng2t** [comment]
Is it convenient for you to tell me how much you set the value of model_devi_f_trust_lo and model_devi_f_trust_hi?
